### DISCUSSION TITLE
Pictures too dark

### DISCUSSION THREAD

**Discussion by Wingstyle** [photoprism/photoprism#2553] [Q&A]
I am wondering if anyone else has had this problem and if so, how did you resolve?

I don't import, I index my pictures into PhotoPrism. 

Not all, but a lot of my photo's are way too dark when viewed in PhotoPrism. The thumbnail as well as when you open up the picture. The original jpg is not dark, only when viewed in PhotoPrism. I can download the picture from PhotoPrism and it looks like the original. 

I have re-indexed and also did a complete rescan and still too dark. In fact some additional pictures are too dark now. I use volume mapping to point  /photoprism/originals and /photoprism/storage to folders on my NAS share and it works well. I renamed the storage folder and made a new one thinking maybe if I had PhotoPrism do a complete scan with no information there, it would generate new thumbnails and they wouldn't be too dark. I did have to reset my password since it didn't have what it needed to allow me to log in with my old credentials. Once I did that I was able to start a rescan. I allowed it to scan for a couple hours and looked at the pictures. Some are fine, but many are too dark. I have over 17K pics and since it wasn't doing what I wanted I cancelled the scan.

Anybody have any suggestions.

**Comment by Wingstyle** [comment]
Here is an example. I used the windows snipping tool to get a screen capture. That is why they are a little bit different in size. 

![original](https://user-images.githubusercontent.com/103340417/181340431-e6daaac1-e76e-4a59-aaa8-ba39b4d08bd7.jpg)

![PPdark](https://user-images.githubusercontent.com/103340417/181340464-886cdef0-d79b-4867-86cd-ea9fb8f9acd9.JPG)

**Reply by lastzero** [reply]
I recommend to use Exiftool to check for any color related metadata such as icc profiles: https://en.m.wikipedia.org/wiki/ICC_profile

**Comment by Wingstyle** [comment]
Thanks for the suggestion. I am familiar with ICC profiles. I process my files with Darktable and somehow I accidentally changed my export profile from sRGB to Linear Rec709 RGB. That was the problem. Now I have to re-export all the pics I made changes to in sRGB and re-index. Thanks again for your help.

**Comment by ps23Rick** [comment]
Hmm.. I’ve got the same thing.. I just imported (via the WebDAV feature) almost 1100 photos — many HEIC and Olympus RAW (ORF) and the images are too dark.  I’ve not done any sort of post-processing of any of these — they came right from the camera and look fine when viewed on my Mac using the standard Apple software. 

Based on the above discussion is sounds like I need to do some sort of post-processing before bringing them into PhotoPrism?  Generally speaking it’s not anything I’ve had to do to this point and if there’s some mostly automated way to do that, I’d love to know!  Thx and I hope I didn’t hijack this discussion..

**Reply by lastzero** [reply]
Could also be an issue with color profiles. If you don't do post processing, there's little sense in using RAW? I'm not even sure if there's a wrong or right for them, since all software we tested renders them different. If you use JPEG, that does not happen, except for unsupported color profiles of course.

**Reply by ps23Rick** [reply]
ok. Thanks.. Maybe I've got more homework to do here.. (aside from Photoprism obviously).. thx!

**Comment by Wingstyle** [comment]
Hi ps23Rick,

Lastzero is correct; there really isn't a reason to use raw unless you do post processing. In fact if you use raw, the in camera processing for jpg is not there so the pictures don't look as good. Usually flat colors etc. My actual issue, figured out by lastzero, was that I did do post processing and when I exported my jpg's I had accidentally changed a setting. Changed that back and everything is fine. 

I don't use a Mac or Olympus, but whatever program you are using to view your raw images probably looks at an embedded jpg and that is what you see. I know some software's actually show the raw file just like it is. PhotoPrism may as well. So when you look at it on your Mac, you are probably looking at the embedded jpg. I hope this helps.

**Reply by ps23Rick** [reply]
Thanks.. In my case I only occasionally use the cameras that put out raw.. one is a drone and yeah I can make it put out jpeg and may just do that if I don't start post-processing.. I just did a quick scan through the dark photos in PhotoPrism and all are either from the drone (in raw) or from the Olympus in raw.  All the HEIC are fine as are any JPG's.. I'll go ahead and do some post processing on the existing raw files and I do appreciate you all for pointing me in the right direction!!  Much appreciated!  And yes, I am liking what I see with PhotoPrism!!  Thanks for the nice work!!

**Reply by Wingstyle** [reply]
A lot of people use Adobe Lightroom for post processing. There is also software's out there like PhotoPrism that you can use for free (donations if you like). In fact PhotoPrism uses a couple of them as raw converters (Darktable and Rawtherapee). You can see this in the advanced tab of settings. I have used Lightroom in the past and looked briefly at Rawtherapee, but am currently using  Darktable on a Windows machine and like it a lot. They do have a pretty steep learning curve but there are Youtube videos to help. There are a couple of others that are easier, less powerful and are also good viewers. FastStone and XnViewMP.
